catalog/src/main/kotlin/com/adevinta/spark/catalog/configurator/component/Component.kt

```diff
@@ -159,6 +159,10 @@ private fun ConfiguratorComponentMenuPreview() {
             .fillMaxSize()
             .wrapContentSize(Alignment.Center),
     ) {
-        ConfiguratorComponentMenu(Components.first(), true) {}
+        ConfiguratorComponentScreen(
+            Components.first(),
+            Configurator("aaaa", "aaaa", "") {},
```

Comment:
What's aaaa", "aaaa" ?

Can you make them into constants?

Reply:
Hum this seems like an unintended change 🤔 
Nice spot!